stock_partner_delivery_window/models/stock_picking.py

```diff
@@ -22,7 +22,7 @@ def _compute_partner_delivery_window_warning(self):
                 continue
 
             anytime_delivery = partner and partner.delivery_time_preference == "anytime"
-            outgoing_picking = self.picking_type_id.code == "outgoing_picking"
+            outgoing_picking = picking.picking_type_id.code != "outgoing"
```

Comment:
Still two things to fix here:
- the `s/self/picking/` fix should land in your IMP commit that transform the onchange into a computed warning msg
- now we are doing `code != "outgoing"`, the variable name should become `not_outgoing_picking` to be meaningful